relion/protocols/protocol_base.py

```diff
@@ -849,76 +887,78 @@ def convertInputStep(self, particlesId, copyAlignment):
         else:
             self.info("In continue mode is not necessary convert the input "
                       "particles")
-        
+
         # if self.realignMovieFrames, self.IS_CLASSIFY must be False.
         if getattr(self, 'realignMovieFrames', False):
             movieParticleSet = self.inputMovieParticles.get()
             movieFn = self._getFileName('movie_particles')
             self.info("Converting set from '%s' into '%s'" %
                       (movieParticleSet.getFileName(), movieFn))
-            
+
             auxMovieParticles = self._createSetOfMovieParticles(suffix='tmp')
             auxMovieParticles.copyInfo(movieParticleSet)
-            
+
             # Discard movie particles that are not present in the
             # refinement set
             for movieParticle in movieParticleSet:
                 particle = imgSet[movieParticle.getParticleId()]
                 if particle is not None:
                     auxMovieParticles.append(movieParticle)
-            
+
             relion.convert.writeSetOfParticles(
                 auxMovieParticles, movieFn, None,
                 fillMagnification=True,
                 postprocessImageRow=self._postprocessImageRow)
+
             mdMovies = md.MetaData(movieFn)
             continueRun = self.continueRun.get()
             continueIter = self._getContinueIter()
-            mdFile = continueRun._getFileName('data', iter = continueIter)
+            mdFile = continueRun._getFileName('data', iter=continueIter)
             mdParts = md.MetaData(mdFile)
-            
+
             self._copyAlignAsPriors(mdParts, em.ALIGN_PROJ)
             mdParts.renameColumn(md.RLN_IMAGE_NAME, md.RLN_PARTICLE_ORI_NAME)
             mdParts.removeLabel(md.RLN_MICROGRAPH_NAME)
-            
+
             mdAux = md.MetaData()
             mdAux.join2(mdMovies, mdParts, md.RLN_PARTICLE_ID,
                         md.RLN_IMAGE_ID, md.INNER_JOIN)
             mdAux.fillConstant(md.RLN_PARTICLE_NR_FRAMES,
                                self._getNumberOfFrames())
-            if relion.Plugin.isVersion2Active():
-                # FIXME: set to 1 till frame averaging is implemented in xmipp
-                mdAux.fillConstant(md.RLN_PARTICLE_NR_FRAMES_AVG, 1)
+
+            # FIXME: set to 1 till frame averaging is implemented in xmipp
```

Comment:
we have to rethink this since now we have relion extr ptcls movies protocol which can do averaging..

Reply:
@delarosatrevin we might ignore this issue, since it is only relevant for relion 2.x old movie processing... what do you think?

Reply:
Yes, I think is is not worth the effort.